storage/volume/core.go

```diff
@@ -76,16 +76,38 @@ func (c *Core) GetVolume(id types.VolumeID) (*types.Volume, error) {
 		},
 	}
 
+	ctx := driver.Contexts()
+
 	// first, try to get volume from local store.
 	obj, err := c.store.Get(id.Name)
 	if err == nil {
```

Comment:
What will happen if err is not nil and err is not `metastore.ErrObjectNotFound` either?
In the code, I think it will ignore this case. Does it make sense? @shaloulcy

Reply:
It means the bolt works not well, for example bucket not found, even bolt file deleted. We should throw the out the error. Remote volume metas are also stored in boltdb